test/unit/LevelPlaneTest.js

```diff
@@ -1057,3 +1057,73 @@ test('weak charge: destroy block', t => {
   t.end();
 });
 
+test('conduit activation success: place block', t => {
+  const data = [
+    '','prismarine','prismarine','prismarine','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','','conduit','','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','prismarine','prismarine','prismarine','prismarine',
+  ];
+  const plane = new LevelPlane(data, 5, 5, null, "actionPlane");
+
+  plane.setBlockAt(new Position(0, 0), new LevelBlock('prismarine'));
+
+  t.deepEqual(plane.getBlockAt(new Position(2, 2)).isActivatedConduit, true);
+
+  t.end();
+});
+
+test('conduit activation fail: place block', t => {
+  const data = [
+    '','prismarine','prismarine','prismarine','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','','conduit','','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','prismarine','prismarine','prismarine','prismarine',
+  ];
+  const plane = new LevelPlane(data, 5, 5, null, "actionPlane");
+
+  plane.setBlockAt(new Position(1, 1), new LevelBlock('prismarine'));
+
+  t.deepEqual(plane.getBlockAt(new Position(2, 2)).isActivatedConduit, false);
+
+  t.end();
+});
+
+test('conduit deactivation success: destroy block', t => {
+  const data = [
+    'prismarine','prismarine','prismarine','prismarine','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','','conduit','','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','prismarine','prismarine','prismarine','prismarine',
+  ];
+  const plane = new LevelPlane(data, 5, 5, null, "actionPlane");
+
+  // Calling this manually since it's normally done by the View
+  plane.resolveConduitState();
+  plane.setBlockAt(new Position(0, 0), new LevelBlock(''));
```

Comment:
I considered leaving the original data empty, and then doing TWO setBlockAt() calls, because the previous tests prove that setting prismarine functions as desired... the double set may be most _honest_ than manually calling `resolveConduitState()`? Because that's not _really_ how that should ever be done?

Thoughts? 🤔

Reply:
I'd probably roll this test into the first one, with two consecutive asserts:

```js
  plane.setBlockAt(new Position(0, 0), new LevelBlock('prismarine'));
  t.deepEqual(plane.getBlockAt(new Position(2, 2)).isActivatedConduit, true);

  plane.setBlockAt(new Position(0, 1), new LevelBlock(''));
  t.deepEqual(plane.getBlockAt(new Position(2, 2)).isActivatedConduit, false);
```

Reply:
For that matter, you could add the "conduit activation fail: place block" test as a third assert.